samples/csharp/mqttClientCertAuthServer/CSharpMqttAuthServer.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <RootNamespace>csharp_server</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.5" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="*" />
```

Comment:
Could we use System.Text.Json as it will be preferred SDK moving forward

Reply:
Actually only `System.Text.Json` is used. I just leave `Newtonsoft.Json` here so that users who want to use `Newtonsoft.Json` can use the code directly....